src/java/org/apache/cassandra/config/Config.java

```diff
@@ -390,6 +390,8 @@ public static class SSTableConfig
     public ParameterizedClass commitlog_compression;
     public FlushCompression flush_compression = FlushCompression.fast;
     public int commitlog_max_compression_buffers_in_pool = 3;
+    public boolean direct_io_for_commitlog_enabled = false;
```

Comment:
what about remove this variable？And if we want to know wether we can use direct ion ,just retun ommitLogDiskAccessMode.direct_io == commitlog_disk_access_mode  
?